### ISSUE TITLE
Implement project selector

### ISSUE THREAD

**Issue by alansemenov** [enonic/app-contentstudio#1373]
When `list` API returns more than one project, there should be a TreeGrid dropdown inside the Content Studio header, similar to the selector we [implemented](https://github.com/enonic/app-contentstudio/issues/671) for layers.

On the screenshot below it shows second level with layers, but in the first version the tree will only have one level (and no icons to expand the nodes). Use the same styling for highlighting current project as we did for the layers. Render project icon if API returns it, otherwise use default icon (which is the Content Tree icon):  ![image](https://user-images.githubusercontent.com/11612490/71727125-9c637a00-2e39-11ea-9a28-66c1205188df.png)

So basically, since project icon is not yet implemented, it will be a flat list with content tree icons.

![image](https://user-images.githubusercontent.com/11612490/71727023-48589580-2e39-11ea-905b-f87a478c677a.png)

**Comment by ashklianko** [comment]
https://github.com/enonic/app-contentstudio/commit/a10b0daca2c88f883bf5f107567e22613fef5e49
